components/chainhook-cli/src/service/mod.rs

```diff
@@ -509,18 +601,21 @@ fn set_predicate_streaming_status(
                     number_of_times_triggered,
                     last_evaluated_block_height,
                 ),
-                PredicateStatus::Expired(ExpiredData {
+                PredicateStatus::ExpiredUnsafe(ExpiredData {
```

Comment:
`Unsafe` could be ambiguous, how about `Expired` + `UnconfirmedExpiration` or `ConfirmedExpiration` + `Expired`?

Reply:
I like it. I went with`UnconfirmedExpiration` +`ConfirmedExpiration` just so they can be consistent and explicit.